doc/cli/vast-pivot.md

```diff
@@ -1,13 +1,24 @@
-The `pivot` command retrieves data of a related type. It inspects each event in
-a query result to find an event of the requested type. If the related type
-exists in the schema, VAST will dynamically create a new query to fetch the
+The `vast pivot` command retrieves data of a related type. It inspects each
+event in a query result to find an event of the requested type. If the related
+type exists in the schema, VAST will dynamically create a new query to fetch the
 contextual data according to the type relationship.
```

Comment:
The first paragraph is still pretty hard to parse without roughly knowing how pivot works already. In particular, the first two sentences give the impression that pivot is creating a subset of the query result containing only events of the requested type. (and even before that one has to make the mental connection between "a related type" and "the requested type")

The third sentence also immediately raises the question of what happens when the related type does not exist in the schema, which is never answered.